sdk/identity/identity/review/identity.api.md

```diff
@@ -365,6 +365,7 @@ export interface TokenCredentialOptions extends CommonClientOptions {
     authorityHost?: string;
     loggingOptions?: LogPolicyOptions & {
         allowLoggingAccountIdentifiers?: boolean;
+        enableSupportLogging?: boolean;
```

Comment:
can we get a changelog item for this as well as perhaps a remark about its existence in the README's logging section?

Reply:
added